middleware/locals.js

```diff
@@ -57,6 +58,44 @@ module.exports = function(req, res, next) {
         };
     });
 
+    /**
+     * Global navigation model
+     */
+    res.locals.globalNavigation = {
```

Comment:
You could possibly simplify this a bit with something like this:

```
const { isString } = require('lodash');

// Nav elements here are either strings
// (eg. sharing the same key for translations as the URL path
// or objects (eg. specifying a custom label/path if they differ)
const navElements = {
    primary: [
        'funding',
        {
            label: 'updates',
            path: '/news'
        },
        'research',
        'contact'
    ],
    secondary: [
      'about',
      'jobs'
    ]
};

const createNavItem = navItem => {
    if (isString(navItem)) {
        return {
            label: req.i18n.__(`global.nav.${navItem}`),
            url: localify(locale)(`/${navItem}`)
        }
    } else {
        return {
            label: req.i18n.__(`global.nav.${navItem.label}`),
            url: localify(locale)(${navItem.path})
        }
    }
};

res.locals.globalNavigation = {
    home: {
        label: req.i18n.__('global.nav.home'),
        url: localify(locale)('/')
    },
    primaryLinks: navElements.primary.map(createNavItem),
    secondaryLinks: navElements.secondary.map(createNavItem)
};
```

(saves on the repeated calls to `localify()` and `req.i18n.__()` boilerplate – though might be a bit too magic?)

Reply:
This feels too magic for me, I think it's good here to be as explicit as possible even if that means a little boilerplate. I've just reduced the `req.i18n.__` noise with a variable though.